pkg/controller.v1alpha3/consts/const.go

```diff
@@ -112,11 +112,14 @@ const (
 	JobKindTF = "TFJob"
 	// JobKindPyTorch is the kind of PyTorchJob.
 	JobKindPyTorch = "PyTorchJob"
+	// JobKindMpi is the kind of MpiJob.
+	JobKindMpi = "MPIJob"
 
 	// built-in JobRoles
 	JobRole        = "job-role"
 	JobRoleTF      = "tf-job-role"
 	JobRolePyTorch = "pytorch-job-role"
+	JobRoleMpi     = "mpi_role_type"
```

Comment:
Yes, refer to 
<https://github.com/kubeflow/mpi-operator/blob/master/pkg/controllers/v1alpha2/mpi_job_controller.go#L81>

Reply:
@terrytangyuan Hi, will we change the label in the future, maybe v1beta1 or v1?

Reply:
Agree with @gaocegege. I think MPI should follow the same patterns as TF Job and Pytorch Job.

Reply:
Thanks. I changed it in https://github.com/kubeflow/mpi-operator/pull/252 for v1 candidate of MPI Operator. Perhaps this PR can add support for v1 candidate directly? The API should be relatively stable even though there isn't official release yet.

Reply:
Thank you @terrytangyuan. I agree, let's support directly v1 MPI Operator in Katib.

Reply:
SGTM

Reply:
OK, actually I have my concern, release branch is always much more stable than dev branch, thus I choose to support the latest release, perhaps I should wait for the next official release for mpi-operator and then submit a new PR? @gaocegege @andreyvelich WDYT

Reply:
I think katib will release new version after MPI operator releases its next version. Thus I think we can safely support v1 to avoid additional work. /cc @johnugeorge

Reply:
The MPI v1 controller and API should be relatively stable now so it's better to just support that.

Reply:
Agree with @gaocegege . 
@YuxiJin-tobeyjin can you change this PR to support v1 MPI Operator version?

Reply:
Ok, but I need to make some tests on latest mpi master branch to ensure changes work as expected, I will do it ASAP.

Reply:
Thanks for your contribution! :tada: :+1:

Reply:
@terrytangyuan  I've tried mpi-operator master branch on our cluster with kubernetes 1.14 and encountered some problems.

First, when creating new mpijob crd ,it throws up

```log
unknown field "additionalPrinterColumns" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceValidation
```

Walk around then, the crd has been created ok, but the mpi-operator is created failed 

```log
the server could not find the requested resource (get mpijobs.kubeflow.org)
```
Maybe problem is here ,still use KubeflowV1alpha2 to get v1 crd?

```url
https://github.com/kubeflow/mpi-operator/blob/acddf3028ce922e24ac3e735a2928ff4487be28f/cmd/mpi-operator.v1/app/server.go#L288
```
Walk around then, but mpi-operator throws 

```log
Failed to list *v1beta1.PodGroup: podgroups.scheduling.volcano.sh
```
So , now use mpi-operator v1 we must use volcano as the bacth scheduler? Since kube-batch latest release do not support v1beta1.PodGroup.  Actually we've already used the kube-batch latest release for some times...

Reply:
Thanks! I fixed the CRD validation issue in https://github.com/kubeflow/mpi-operator/pull/257#issuecomment-628612127. Yes Volcano has better support and maintenance. You may want to update your MPI Operator's deployment yaml which includes permissions for Volcano resources: https://github.com/kubeflow/mpi-operator/blob/master/deploy/v1/mpi-operator.yaml